doc/source/whatsnew/v1.3.0.rst

```diff
@@ -134,6 +134,30 @@ a copy will no longer be made (:issue:`32960`)
 The default behavior when not passing ``copy`` will remain unchanged, i.e.
 a copy will be made.
 
+Centered Datetime-Like Rolling Windows
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+When performing rolling calculations on :class:`DataFrame` and :class:`Series`
+objects with a datetime-like index, a centered datetime-like window can now be
+used (:issue:`38780`).
+For example:
+
+.. ipython:: python
+
+   In [1]: df = DataFrame(
+              {"A": [0, 1, 2, 3, 4]},
+              index=date_range("2020", periods=5, freq="1D")
+           )
+
+   In [2]: df.rolling("2D", center=True).mean()
```

Comment:
Don't need the `In` `Out` annotations when using `.. ipython:: python` (the code will execute line by line